src/main/java/org/opentripplanner/updater/alerts/GtfsEnhancedRealtimeAlertsUpdater.java

```diff
@@ -77,21 +77,31 @@ public String toString() {
         JsonNode json = mapper.readTree(jsonString);
 
         for (JsonNode node: json.get("entity")) {
+            if (!node.hasNonNull("alert")) {
+                continue;
+            }
+
+            JsonNode alert = node.get("alert");
```

Comment:
can this and the statements above be combined to make the code more readable and a little more efficient?
```
JsonNode alert = node.get("alert");
if (alert == null)
  continue;

Reply:
Done